### PR TITLE
Integrate standa stages into catkit framework

### PR DESCRIPTION
**Pull request by jamienoss**
Signed-off-by: James Noss <jnoss@stsci.edu>

### CODE REVIEW

File: catkit/hardware/standa/stages.py [hunk 1]
```diff
@@ -5,99 +5,89 @@
...
+from catkit.interfaces.Instrument import Instrument
+
 
 cur_dir = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
 os.chdir(cur_dir)
 ximcDir = (f'{cur_dir}/ximc-2.13.3/ximc')
 ximcPackageDir = os.path.join(ximcDir, "crossplatform", "wrappers", "python")
 sys.path.append(ximcPackageDir)
-from pyximc import *  # noqa: E402
+import pyximc  # noqa: E402
```
Comment (jamienoss) [inline catkit/hardware/standa/stages.py]: https://github.com/aidancgray/idg-stuf/blob/main/stages/ximc-2.13.3/ximc/crossplatform/wrappers/python/pyximc.py

File: catkit/hardware/standa/stages.py [hunk 2]
```diff
@@ -5,99 +5,98 @@
...
         self.u_homeOffset, self.homeOffset = math.modf(homeOffset)
         self.conversionFactor = conversionFactor
         self.units = units
 
-        self.stageDev = self.lib.open_device(deviceID)
+    def _open(self):
+        return self.instrument_lib.lib.open_device(self.deviceID)
+
+    def _close(self):
+        self.instrument_lib.lib.close_device(ctypes.byref(self.instrument))
```
Comment (jamienoss) [inline catkit/hardware/standa/stages.py:53]: API: https://github.com/aidancgray/idg-stuf/blob/main/stages/ximc-2.13.3/ximc/win64/wrappers/csharp/ximcnet.cs#L1296

**Comment by jamienoss** [comment]
@aidancgray Can you take a look over this and check that I didn't mess something up, please, thanks.

**Review by aidancgray — APPROVED** [review]
Looks good!

**Pull request merged**